ports/wxcharts/vcpkg.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "wxcharts",
+  "version": "0.1.0",
```

Since wxcharts hasn't been released yet, this should probably be a `version-date`:

```suggestion
  "version-date": "2022-04-22",
```